src/main/java/io/kestra/plugin/aws/s3/ActionInterface.java

```diff
@@ -7,20 +7,21 @@
 
 public interface ActionInterface {
     @Schema(
-        title = "The action to perform on the retrieved files."
+        title = "The action to perform on the retrieved files. If using 'NONE' make sure to do some actions on the files to avoid infinite triggering."
```

Comment:
```suggestion
        title = "The action to perform on the retrieved files. If using 'NONE', make sure to handle the files inside your flow to avoid infinite triggering."
```